libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -121,11 +121,16 @@ export class ExecuteBridgeRequest {
       timeout: DEFAULT_TIMEOUT,
       json: command.event,
       retry: {
-        calculateDelay: ({
-          attemptCount,
-        }: {
-          attemptCount: number;
-        }): number => {
+        calculateDelay: ({ attemptCount, computedValue }) => {
+          if (computedValue === 0) {
```

Comment:
This shouldn't happen in our case because we don't reduce the delay or use the 'limit' option, but it's a nice safeguard for the future.

Reply:
It does happen, because we supply the `statusCodes` and `errorCodes` options. The lack of this check was causing the `statusCodes` and `errorCodes` configuration to be ignored, resulting in 3 retries always occurring. There's no need to retry a `400 Bad Request` response, for example. This configuration ensures that such responses are not retried anymore.

I will add a code comment to clarify this.

Reply:
I added a comment, let me know if that provides more clarity.

Reply:
It ends up being overwhelming as we often need to opt out of everything that comes out of the box. We need more control and an opt-in approach. It's not actionable for now, I just wanted to share my point of view.